builds/misc/templates/image-linux-rust-jobs.yaml

```diff
@@ -1,4 +1,4 @@
-# archname is used for job/display name
+# archname is used for job/display name and get image
```

Comment:
I didn't see any other change in this file, why update this comment?

Reply:
Oh, this is a mistake.